packages/modules/payment/src/loaders/index.ts

```diff
@@ -1,2 +1 @@
 export * from "./providers"
-export * from "./defaults"
```

Comment:
Moved the code inside `providers` since it didn't load defaults, but instead it just wrote to the DB. This is more in-line with other loaders.